include/triton/Dialect/TritonGPU/IR/TritonGPUOps.td

```diff
@@ -129,6 +129,40 @@ def TTG_AsyncCopyGlobalToLocalOp : TTG_Op<"async_copy_global_to_local", [
   }];
 }
 
+def TTG_AsyncCopyLocalToGlobalOp : TTG_Op<"async_copy_local_to_global", [
```

Comment:
Should this be in amdgpu? AFAIK there isn't an equivalent in nvidia hardware.

Reply:
Oh if that's the case then yeah we can move to amdgpu specific.